docs/quickstart.md

```diff
@@ -169,85 +162,26 @@ Confirm password:
 
 v1.2.0
 
-AEA configurations successfully initialized: {'author': 'fetchai'}
+AEA configurations successfully initialized: {'author': 'valory'}
 ```
 
 <div class="admonition note">
   <p class="admonition-title">Note</p>
   <p>If you would rather not create an account on the registry at this point, then run <code>aea init --local</code> instead.</p>
 </div>
 
+
+
 ## Echo skill demo
```

Comment:
For now let's ask the user to copy the packages folder from the repo via `svn export` so they can use `--local`. We don't have a remote registry atm.

Reply:
Is this needed? I ran this locally and didnt have any issues. I can confirm on a new machine, and the command to svn pull is;
```
svn checkout https://github.com/valory-xyz/open-aea/trunk/packages packages```

Reply:
Yes!